docs/querying/basics.md

```diff
@@ -204,11 +204,47 @@ The same works for range vectors. This returns the 5-minute rate that
 
     rate(http_requests_total[5m] offset 1w)
 
+### @ modifier
+
+The `@` modifier allows changing the time for individual instant
```

Comment:
What does "changing the time" mean? Instant and range vectors don't have a time.